src/promptflow/promptflow/_constants.py

```diff
@@ -64,6 +64,7 @@ class AvailableIDE:
 # trace related
 OTEL_RESOURCE_SERVICE_NAME = "promptflow"
 DEFAULT_SPAN_TYPE = "default"
+RUNNING_LINE_RUN_STATUS = "Running"
```

Comment:
may wait for UX changes in this PR, otherwise, users will hit the "No span data" issue.